### DISCUSSION TITLE
Welcome to taskcluster Discussions!

### DISCUSSION THREAD

**Discussion by petemoore** [taskcluster/taskcluster#5034] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Note, most day-to-day chat tends to happen on [matrix](https://wiki.mozilla.org/Matrix), but if you are not a matrix user, feel free to join us on Discussions.

**Comment by Aitools-pixel** [comment]
Hey taskcluster im saad im in need of marketer i develop professional websites but dont know how to sell it if u have any possiblities so u can tell me otherwise im also ready to work under a large companies to byild websites for them
